mmv1/third_party/terraform/services/securitycentermanagement/resource_scc_management_organization_event_threat_detection_custom_module_test.go.erb

```diff
@@ -72,7 +72,10 @@ func testAccSecurityCenterManagementOrganizationEventThreatDetectionCustomModule
 
 func testAccSecurityCenterManagementOrganizationEventThreatDetectionCustomModule__sccOrganizationCustomModuleExample(context map[string]interface{}) string {
 	return acctest.Nprintf(`
+<% unless version == 'ga' -%>
```

Comment:
Can you move `<% unless version == 'ga' -%>` to the top of this file as what is done for the other three files?

Reply:
Addressed.